mindjunkies/forums/views.py

```
@@ -177,3 +178,65 @@ def post(self, request, *args, **kwargs):
+    """View for toggling likes on Comment objects"""
+    model = ForumComment
+    template_name = 'forums/partials/like_comment.html'
+    context_object_name = 'comment'
+
+
+class LikeReplyView(LikeToggleView):
+    """View for toggling likes on Reply objects"""
+    model = Reply
+    template_name = 'forums/partials/like_reply.html'
+    context_object_name = 'reply'
+      
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)